pkg/summarizer/summary.go

```diff
@@ -101,18 +101,30 @@ type Fixer func(context.Context, *config.DashboardQueue) error
 // Will use concurrency go routines to update dashboards in parallel.
 // Setting dashboard will limit update to this dashboard.
 // Will write summary proto when confirm is set.
-func Update(ctx context.Context, client gcs.ConditionalClient, mets *Metrics, configPath gcs.Path, concurrency int, tabPathPrefix, summaryPathPrefix string, allowedDashboards []string, confirm bool, features FeatureFlags, freq time.Duration, fixers ...Fixer) error {
+
+type UpdateOptions struct {
```

Comment:
According to Go style:  this is an exported struct, as such, it's advised to have a comment describing what this is used for.
`UpdateOptions aggregates the ....`